contract_price_revision/tests/test_contract_price_revision.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 Tecnativa - Vicent Cubells
+# Copyright 2019 Tecnativa - Carlos Dauden
+# Copyright 2020 ACSONE SA/NV
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.addons.contract.tests.test_contract import TestContractBase
+
+
+class TestContractPriceRevision(TestContractBase):
+    def _create_wizard(self, v_type="percentage", value=0.0):
+        # TODO: Limitation here, start date should be on the
+        # beginning of next period (should not have a gap)
+        self.wizard = self.env["contract.price.revision.wizard"].create(
+            {
+                "date_start": "2018-02-01",
+                "variation_type": v_type,
+                "variation_percent": value,
+                "fixed_price": value,
+            }
+        )
+
+    def execute_wizard(self):
+        self.wizard.with_context(active_ids=self.contract.id).action_apply()
```

Comment:
`active_ids` should be a `list`.

Ideally, the model should be specified too:
`active_ids = self.contract.ids, active_model=self.contract._name`